### ISSUE TITLE
Migrate the Weebly site to the new platform

### ISSUE THREAD

**Issue by eljobe** [wmwnc/website#2]
It shouldn't take long to move the pages at the current place to the new home

**Comment by eljobe** [comment]
@mgermain99 I have moved over the content from http://womensmarch.weebly.com to https://womensmarch.com

The only thing that is a little different is the way the `About` menu works. On the weebly site, the word `About` was not a link to a page, but rather just a menu title and all the pages were nested under `About`. I haven't figured out how to do this same thing in WordPress, so the `About` page is a real page and I moved the content from the `Mission and Vision` page to the `About` page. I hope that doesn't cause too much confusion. But, if we need to make it work the previous way, I can modify it further.

I also did add a contact form on the `Contact Us` page. I think if a visitor fills it out, it will email both of us. I can configure it to a different email address if you prefer.
